<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>786</width>
    <height>477</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>310</y>
      <width>256</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Patch Version to use</string>
    </property>
    <widget class="QComboBox" name="patchSelector">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>241</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="patchButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>241</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Patch</string>
     </property>
    </widget>
   </widget>
   <widget class="QGraphicsView" name="modPreview">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>30</y>
      <width>256</width>
      <height>256</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>256</width>
      <height>0</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="browseButton">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>20</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse...</string>
    </property>
   </widget>
   <widget class="QLabel" name="selectedDirectoryLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>291</width>
      <height>21</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border: 1px solid black;
color: rgb(100,100,100)</string>
    </property>
    <property name="text">
     <string>Mods location</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>10</y>
      <width>20</width>
      <height>411</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QListWidget" name="activeModList">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>291</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="inactiveModList">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>280</y>
      <width>291</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="removeModButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>230</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Add to Active&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(122, 225, 165);</string>
    </property>
    <property name="text">
     <string>⮝</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addModButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>230</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Remove from Active&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(243, 159, 149);</string>
    </property>
    <property name="text">
     <string>⮟</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>ACTIVE</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>260</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>INACTIVE</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>786</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="sizeGripEnabled">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
